task-sdk/tests/task_sdk/log/test_log.py

```diff
@@ -112,7 +111,7 @@ def test_logs_are_masked(captured_logs):
     secrets_masker = SecretsMasker()
     secrets_masker.add_mask("password")
     with mock.patch(
```

Comment:
WAT. Not a change in this Pr, but why are we mocking the redact function, rather than.... letting it run?

Reply:
Umm yeah, will clean it up in a follow up